R/parse.R

```diff
@@ -116,7 +107,8 @@ coerce_datetime <- function(x, to, ...) {
   } else if (is.numeric(x)) {
     vctrs::new_datetime(as.double(x), tzone = tzone(to))
   } else if (is.character(x)) {
-    as.POSIXct(x, tz = tzone(to))
+    # Parse as ISO8601
+    as.POSIXct(strptime(x, format = "%Y-%m-%dT%H:%M:%SZ"), tz = tzone(to))
```

Comment:
Nothing you did here but 🫠 at the two different time parsing/formatting functions. But that's just how it is

Reply:
Yeah, _as it turns out_ strptime returns POSIXlt

Reply:
🫠